ruby/introduction/installing_ruby.md

```diff
@@ -243,6 +262,14 @@ rbenv install 3.1.2 --verbose
 
 This command will take 10-15 minutes to complete. The `--verbose` flag will show you what's going on so you can be sure it hasn't gotten stuck. While it installs, take this time to watch [this video](https://www.youtube.com/watch?v=X2CYWg9-2N0) or to get a glass of water.
 
+#### Updating a Previous Install
```

Comment:
Same issue with the subheadings as the first. Again, perhaps the answer is to just upgrade `ruby-build` by default?

Reply:
> Again, perhaps the answer is to just upgrade ruby-build by default?

I totally agree. It seemed like the flow would be better if the process was explicit, but the way I had arranged the portions actually interrupts the flow in an unanticipated way.